src/endpoints/merge.rs

```diff
@@ -0,0 +1,16 @@
+use crate::papers::Merger;
+use crate::prelude::*;
+use futures::{FutureExt, TryFutureExt};
+
+pub(crate) async fn merge(merge_spec: MergeSpec, config: Arc<Config>) -> Result<Response, EndpointError> {
+    merge_spec.validate()?;
+
+    tokio::executor::spawn(
```

Comment:
Why is there no await?

Reply:
`spawn` spawns a background task. When we submit/merge we send a 200 response immediately (just validating the payload), and run the task in the background. Running the task and `await`ing here would mean we wait for the pdf generation to be complete before sending the response.

Reply:
ah ok, makes sense!